### ISSUE TITLE
AppBar: Keyboard focus rectangle is flaky

### ISSUE THREAD

**Issue by rigdern** [winjs/winjs#511]
When clicking on a AppBar command and then arrowing around, the keyboard focus rectangle is invisible the first time focus returns to the item that was clicked on.

This is not due to mouse hover so this bug is different than #510.
1. Open http://try.buildwinjs.com/#appbar
2. Click on "Edit" to give it focus.
3. Move the mouse cursor away from the AppBar to avoid triggering mouse hover.
4. Hit the right arrow to move focus to "Add"
5. Hit the left arrow to move focus back to "Edit"

Expected: "Edit" has a white keyboard focus rectangle around it
![image](https://cloud.githubusercontent.com/assets/199935/3883028/2dbe3922-219d-11e4-9c5e-9386f09a99b7.png)

Actual: "Edit" doesn't have a keyboard focus rectangle
![image](https://cloud.githubusercontent.com/assets/199935/3883033/3a78e9be-219d-11e4-8596-3826710ca7a1.png)

If you repeat steps 4 and 5, "Edit's" keyboard focus rectangle will start rendering again.

Repros in Chrome (Mac/Win) and Safari. Doesn't repro in IE11.
